### PR TITLE
[CHORE]migrate gradle settings and build script to kotlin

### PR DESCRIPTION
**Pull request by laiyin-xue**
- update gradle to 5.6.1
- update related dependencies versions

for issue #1[](https://github.com/ekino/gradle-docker-plugin/issues/1)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+group = "com.ekino.oss.gradle.plugin"
+description = "Gradle Plugin for Docker management"
+version = "0.0.1-SNAPSHOT"
+
+val dockerComposePluginVersion = "0.9.4"
+val junitVersion = "5.5.1"
+val ossrhUsername = ""
+val ossrhPassword = ""
+val ossrhUrl = ""
```
Comment (clemstoquart) [inline build.gradle.kts]: I don't think it's going to work this way. You should have something like that :
```kotlin
repositories {
    maven {
      val ossrhUrl: String? by project
      val ossrhUsername: String? by project
      val ossrhPassword: String? by project

      url = uri(ossrhUrl ?: "")

      credentials {
        username = ossrhUsername
        password = ossrhPassword
      }
    }
  }
```

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
+#Wed Aug 28 17:12:58 CEST 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.1-bin.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```
Comment (clemstoquart) [inline gradle/wrapper/gradle-wrapper.properties]: It shouldn't have add this line. How did you perform the wrapper upgrade ?

Comment (laiyin-xue) [reply gradle/wrapper/gradle-wrapper.properties]: for me, this line is not brought by the wrapper upgrade and it is generated when i compiled this project with his previous version.

Comment (laiyin-xue) [reply gradle/wrapper/gradle-wrapper.properties]: i will rerun the commande `./gradlew wrapper --gradle-version=5.6.1`

Comment (laiyin-xue) [reply gradle/wrapper/gradle-wrapper.properties]: fixed

File: build.gradle.kts (hunk 1 above)
Comment (laiyin-xue) [reply build.gradle.kts]: fixed

**Pull request merged**